### PR TITLE
Post to Buzz

### PR DESCRIPTION
**Pull request by jgondron**
Adds an action that will create the media object on Buzz. This will typically be called once the user has finished uploading to the designated s3 location. To do this I added a service class that will ideally handle all of the communication to the Buzz server and refactored the save_media_thumbnail service to use this instead of directly handling this communication.

### CODE REVIEW

**Comment by rfox2** [comment]
👍

File: app/services/save_media_thumbnail.rb [hunk 1]
```diff
@@ -32,19 +32,11 @@ def send_image_request
...
-    if response.success?
-      response
-    else
-      false
-    end
-  end
-
   def update_media_record
-    @media_response = send_media_request
+    @media.thumbnail_url = @image_response.body["contentUrl"]
```
Comment (jonhartzler) [inline app/services/save_media_thumbnail.rb]: Isn't content URL the full size image?

**Comment by jonhartzler** [comment]
One question then 👍

File: app/services/save_media_thumbnail.rb (hunk 1 above)
Comment (jgondron) [inline app/services/save_media_thumbnail.rb]: Yes, this was done as part of this change: https://github.com/ndlib/honeycomb/commit/f2bc0893cd1514f1f55582b74a81519065e9b7be#diff-771f53e3b0e5494ce9cb7ef68d923165R83. I kept that same behavior when refactoring. Did you or Rob discuss using the full image?

**Pull request merged**